react-backend/client/src/api/index.js

```diff
@@ -244,6 +244,15 @@ export const CbAdmin = {
       },
     ),
 
+  getFeedback: (tkn, since, until) =>
+    axios.get(
+      '/api/cb/feedback',
+      { params: {
```

Comment:
Axios adds [query strings on get requests using `params:`](https://github.com/axios/axios/blob/master/README.md#request-config) and [you can access them with express on `req.query`](http://expressjs.com/en/api.html#req.query). If it's sent under the key `query`, you would have to access it in express as `req.query.query` 😩

Reply:
I was more talking about using the `filter` field, since what you're doing is filtering feedback by date. No point having generic query fields if we don't use them.

Reply:
> No point having generic query fields if we don't use them.

What do you mean by this?